frontend/www/js/omegaup/arena/contest_list.js

```diff
@@ -1,102 +1,33 @@
-omegaup.arena = omegaup.arena || {};
+import { API, UI, OmegaUp, T } from '../omegaup.js';
+import Vue from 'vue';
+import arena_ContestList from '../components/arena/ContestList.vue';
 
-// TODO: This really should be a Knockout component.
-omegaup.arena.ContestList = function(domElement, apiParams, uiParams) {
-  var self = this;
-  self.domElement = domElement;
-
-  var actualApiParams = $.extend(
-    {
-      active: 'ALL',
-      recommended: 'ALL',
-      // TODO: Make this match uiParams.pageSize and do smaller requests.
-      page_size: 1000,
+OmegaUp.on('ready', function() {
+  Date.setLocale(omegaup.T.locale);
+  const payload = JSON.parse(document.getElementById('payload').innerText);
+  for (const timeType in payload.contests) {
```

Comment:
esto funciona?

```suggestion
  for (const [timeType, contests] of Object.entries(payload.contests)) {
```

?

Reply:
Creo que si funciona